src/Twitter.php

```diff
@@ -313,10 +322,10 @@ private function formatResponse(Response $response, string $format)
             case self::RESPONSE_FORMAT_JSON:
                 return $body;
             case self::RESPONSE_FORMAT_ARRAY:
-                return json_decode($body, true);
+                return json_decode($body, true, 512, JSON_THROW_ON_ERROR);
```

Comment:
I guess for this we'll need to bump the min version to 7.3 (or maybe use a polyfill)?

Reply:
Yes. I'd prefer we bump the min version